build_test/pubspec.yaml

```
@@ -4,26 +4,31 @@ version: 1.3.6
 homepage: https://github.com/dart-lang/build/tree/master/build_test
 
 environment:
-  sdk: ">=2.9.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
   async: ">=1.2.0 <3.0.0"
```

We should up all deps to require the null safe versions (we are likely relying on them in practice)

```suggestion
  async: ^2.5.0
```